src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -938,6 +938,66 @@ public static partial class Kernel32
+        /// If this parameter is NULL, it returns a handle to the file used to create the calling process (.exe file).
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is a handle to the specified module.
+        /// If the function fails, the return value is NULL.To get extended error information, call GetLastError.
+        /// </returns>
+        /// <remarks>
+        /// This function does not retrieve handles for modules that were loaded using the <see cref="LoadLibraryExFlags.LOAD_LIBRARY_AS_DATAFILE"/> flag.
+        /// This function returns a handle to a mapped module without incrementing its reference count.
+        /// However, if this handle is passed to the <see cref="FreeLibrary"/> function, the reference count of the mapped module will be decremented.
+        /// Therefore, do not pass a handle returned by this function to the <see cref="FreeLibrary"/> function. Doing so can cause a DLL module to be unmapped prematurely.
+        /// </remarks>
+        [DllImport(nameof(Kernel32), SetLastError = true, CharSet = CharSet.Auto)]
+        public static extern SafeLibraryHandle GetModuleHandle(string lpModuleName);
+
+        /// <summary>
+        /// Retrieves a module handle for the specified module and increments the module's reference count unless <see cref="GetModuleHandleExFlags.GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT"/> is specified.
+        /// The module must have been loaded by the calling process.
+        /// </summary>
+        /// <param name="dwFlags">
+        /// This parameter can be zero or one or more of the following values.
+        /// If the module's reference count is incremented, the caller must use the <see cref="FreeLibrary"/> function to decrement the reference count when the module handle is no longer needed.</param>
+        /// <param name="lpModuleName">The name of the loaded module (either a .dll or .exe file), or an address in the module (if <paramref name="dwFlags"/> is <see cref="GetModuleHandleExFlags.GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS"/>).</param>
+        /// <param name="phModule">
+        /// A handle to the specified module. If the function fails, this parameter is NULL.
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, returns true. If the function fails, the returns false.
+        /// To get extended error information, see <see cref="GetLastError"/>.
+        /// </returns>
+        /// <remarks>
+        /// The handle returned is not global or inheritable. It cannot be duplicated or used by another process.
+        /// This function does not retrieve handles for modules that were loaded using the <see cref="LoadLibraryExFlags.LOAD_LIBRARY_AS_DATAFILE"/> flag.
+        /// </remarks>
+        [DllImport(nameof(Kernel32), SetLastError = true, CharSet = CharSet.Auto)]
```

Comment:
Can you use `CharSet.Unicode` ? We'll do an unification pass at some point so no problem if it's only for following PRs the 2 are equivalent but it would be clearere.